### PR TITLE
Create Software quickstart guide

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/1766

### CODE REVIEW

File: software/documentation-overview.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+Depending on your organization's needs ,use the following docs to complete additional, optional configuration during your installation. 
+
+- [Generate a self-signed certificate](self-signed-certificate.md)
+- [Renew TLS certificates on Astronomer Software](renew-tls-cert.md)
+- [Install Astronomer on an airgapped system](install-airgapped.md)
+
+## Key DAG author actions
+
+If you regularly write and deploy DAGs, use the following docs as your core reference:
```
Comment (jwitz) [inline software/documentation-overview.md:26]:
```suggestion
If you primarily write and deploy DAGs, use the following docs as your core reference:
```

File: software/documentation-overview.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+- [Log in to Astronomer Software](log-in-to-software.md)
+- [Create an Astro project](create-project.md)
+- [Deploy code using the Astro CLI](deploy-cli.md)
+- [Set environment variables](environment-variables.md)
+- [Upgrade Airflow](manage-airflow-versions.md)
+
+## Key administrator actions
+
+If you need to regularly manage users and cloud resources for Astronomer Software, use the following docs as your core reference:
```
Comment (jwitz) [inline software/documentation-overview.md:36]:
```suggestion
If you regularly manage users and cloud resources for Astronomer Software, use the following docs as your core reference:
```

File: software/documentation-overview.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+- [Apply a platform configuration changes](apply-platform-config.md)
+- [Manage users on Astronomer Software](workspace-permissions.md)
+- [Manage and customize user roles](manage-platform-users.md)
+- [Alerting in Astronomer Software](platform-alerts.md)
+- [Configure a Deployment on Astronomer Software](configure-deployment.md)
+- [Upgrade Astronomer](upgrade-astronomer.md)
+
+## Keep up to date on releases and compatibility
+
```
Comment (jwitz) [inline software/documentation-overview.md:46]:
```suggestion

The following docs are frequently updated with the latest release and compatibility information: 

```

File: software/documentation-overview.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
+---
+sidebar_label: 'Documentation overview'
+title: 'Astronomer Software documentation overview'
+id: documentation-overview
+description: A list of all key documents for using and maintaining Astronomer Software.
```
Comment (kwcanuck) [inline software/documentation-overview.md:5]:
```suggestion
description: An overview of the documentation available to install and manage Astronomer Software.
```

File: software/documentation-overview.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
+---
+sidebar_label: 'Documentation overview'
+title: 'Astronomer Software documentation overview'
```
Comment (kwcanuck) [inline software/documentation-overview.md:3]:
```suggestion
title: 'Astronomer Software documentation resources'
```

File: software/documentation-overview.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
+---
+sidebar_label: 'Documentation overview'
```
Comment (kwcanuck) [inline software/documentation-overview.md:2]:
```suggestion
sidebar_label: 'Documentation reference'
```

File: software/documentation-overview.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+title: 'Astronomer Software documentation overview'
+id: documentation-overview
+description: A list of all key documents for using and maintaining Astronomer Software.
+---
+
+Use this page to quickly access the most important docs for using and maintaining Astronomer Software.
```
Comment (kwcanuck) [inline software/documentation-overview.md:8]:
```suggestion

```
If you want to keep this statement, use it as an introduction that appears below the heading.

Comment (kwcanuck) [inline software/documentation-overview.md:8]:
```suggestion
Use this reference to identify the documentation resources that are available to help you install and manage Astronomer Software.
```

File: software/documentation-overview.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+
+## Install Astronomer Software 
+
+Use the following guides as the core reference for configuring a new installation of Astronomer Software
```
Comment (kwcanuck) [inline software/documentation-overview.md:12]:
```suggestion
The following guides will help you install and configure a new Astronomer Software installation: 
```

File: software/documentation-overview.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+
+- [Install Astronomer on AWS](install-aws.md)
+- [Install Astronomer on GCP](install-gcp.md)
+- [Install Astronomer on Azure](install-azure.md)
+
+Depending on your organization's needs ,use the following docs to complete additional, optional configuration during your installation.
```
Comment (kwcanuck) [inline software/documentation-overview.md:18]:
```suggestion
The following topics are recommended when your organization needs optional configurations during the Astronomer Software installation: 
```

File: software/documentation-overview.md [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+- [Install Astronomer on GCP](install-gcp.md)
+- [Install Astronomer on Azure](install-azure.md)
+
+Depending on your organization's needs ,use the following docs to complete additional, optional configuration during your installation. 
+
+- [Generate a self-signed certificate](self-signed-certificate.md)
+- [Renew TLS certificates on Astronomer Software](renew-tls-cert.md)
+- [Install Astronomer on an airgapped system](install-airgapped.md)
+
+## Key DAG author actions
```
Comment (kwcanuck) [inline software/documentation-overview.md:24]:
```suggestion
## Author DAGs
```

File: software/documentation-overview.md [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+
+If you primarily write and deploy DAGs, use the following docs as your core reference:
```
Comment (kwcanuck) [inline software/documentation-overview.md:26]:
```suggestion
The following topics are intended to help you write and deploy DAGs:
```

File: software/documentation-overview.md [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
+
+- [Log in to Astronomer Software](log-in-to-software.md)
+- [Create an Astro project](create-project.md)
+- [Deploy code using the Astro CLI](deploy-cli.md)
+- [Set environment variables](environment-variables.md)
+- [Upgrade Airflow](manage-airflow-versions.md)
+
+## Key administrator actions
```
Comment (kwcanuck) [inline software/documentation-overview.md:34]:
```suggestion
## Administer Astronomer Software
```

File: software/documentation-overview.md [hunk 13]
```diff
@@ -0,0 +1,51 @@
...
+
+If you regularly manage users and cloud resources for Astronomer Software, use the following docs as your core reference:
```
Comment (kwcanuck) [inline software/documentation-overview.md:36]:
```suggestion
If you're an Astronomer Software administrator, the following topics can help you get started:
```

File: software/documentation-overview.md [hunk 14]
```diff
@@ -0,0 +1,51 @@
...
+
+- [Apply a platform configuration changes](apply-platform-config.md)
+- [Manage users on Astronomer Software](workspace-permissions.md)
+- [Manage and customize user roles](manage-platform-users.md)
+- [Alerting in Astronomer Software](platform-alerts.md)
+- [Configure a Deployment on Astronomer Software](configure-deployment.md)
+- [Upgrade Astronomer](upgrade-astronomer.md)
+
+## Keep up to date on releases and compatibility
```
Comment (kwcanuck) [inline software/documentation-overview.md:45]:
```suggestion
## Keep up to date
```

File: software/documentation-overview.md [hunk 15]
```diff
@@ -0,0 +1,51 @@
...
+
+The following docs are frequently updated with the latest release and compatibility information:
```
Comment (kwcanuck) [inline software/documentation-overview.md:47]:
```suggestion
The following topics are available to help make sure you're aware of the latest Astronomer Software functionality and what's required to run Astronomer Software: 
```

**Review by kwcanuck — APPROVED** [review]

**Review by frankcash — APPROVED** [review]

**Pull request merged**